src/js/common/components/RequiredLoginView.jsx

```diff
@@ -141,17 +99,17 @@ class RequiredLoginView extends React.Component {
       view = <LoadingIndicator setFocus message="Loading your information"/>;
     } else {
       if (this.props.authRequired === 1) {
-        if (this.state.accountType >= 1) {
+        if (this.props.userProfile.accountType >= 1) {
```

Comment:
Unrelated to your PR, can we have enum constants for readability instead of magic numbers?  For example, I dont know what an `authRequired == 1` or `accountType >=1` means.